pulp/overlays/moc/smaug/opf-pulp-route.yaml

```diff
@@ -1,15 +0,0 @@
----
```

Comment:
we don't need the pulp-web anymore, the operator will build the routes automatically

Reply:
Will the certs be pulled?

Reply:
good point, I'm not sure, @git-hyagi do you know?

Reply:
ack, shall we keep the route till we get that pr in.
in meantime, we can get the rest of the changes w.r.t to the CRs into cluster
it would balance the current deployment.

Reply:
we don't have a single route now, the operator creates multiple roles for different paths, so even if we keep this file it won't work